v2/azure-arm.yaml

```diff
@@ -1016,6 +1021,29 @@ objectModelConfiguration:
       FlexibleServers_FirewallRule:
         $exportAs: FlexibleServersFirewallRule
         $supportedFrom: v2.0.0-alpha.2
+    2022-01-20-preview:
+      FlexibleServer:
+        $export: true
+        $supportedFrom: v2.0.0-beta.4
+        $azureGeneratedSecrets:
+          - FullyQualifiedDomainName
```

Comment:
Is this really a secret? Shouldn't this be exported as a config map instead?

Reply:
I matched it in from `2021-06-01`. Although, it should be a configmap. I'm thinking, we have other resources(dbformariadb, etc) as well for which we're doing the same. So maybe all at once, I can change them to be exported to a configmap?

Reply:
As discussed, offline, we have a number of older resources that publish non-sensitive information as secrets because that was the only mechanism we had at the time. We don't want to modify those, because it would be a needless breaking change.

New resources (including new versions of existing resources) should do things properly:
* Sensitive information (e.g. api keys, Azure generated passwords, etc) should be made available to secrets.
* All other information (e.g. domain names etc) should be made available as config maps.